### PR TITLE
Feat/add podspec

### PR DESCRIPTION
**Pull request by nicolai86**
this PR adds a podspec to the project

### CODE REVIEW

File: react-native-xmpp.podspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name         = "react-native-xmpp"
+  s.version      = "0.2.1"
+  s.license      = "MIT"
+  s.homepage     = "https://github.com/aksonov/react-native-xmpp"
+  s.authors      = { 'Marc Shilling' => 'marcshilling@gmail.com' }
```
Comment (Froelund) [inline react-native-xmpp.podspec:6]: Ehm - Who's this?

Comment (nicolai86) [inline react-native-xmpp.podspec:6]: Copypasta ^^

Comment (nicolai86) [inline react-native-xmpp.podspec:6]: I was taking a step at adding a podspec so you can include this dependency more easily in a cocoapods managed project, but it seems to require quite a lot additional changes, sadly

Comment (aksonov) [inline react-native-xmpp.podspec:6]: I don't understand why we need cocoa pods at all. rnpm works fine and it is expected way to work with Native components and React Native.

**Comment by nicolai86** [comment]
the PR was opened by accident against upstream.

**Pull request closed without merging**